simulation-ros/src/turtlebot2i/turtlebot2i_safety/developing_scripts/get_robot_pyhsical_limit.md

```diff
@@ -0,0 +1,22 @@
+# This code aims to get some important parameters of robot.
```

Comment:
either rename to README or move under

```
"""
Python docstrings in the script itself
"""
```

see https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring